FirebaseRemoteConfig/Tests/Unit/FIRRemoteConfigComponentTest.m

```diff
@@ -167,6 +167,35 @@ - (void)testThrowsWithNilGCMSenderID {
   XCTAssertThrows([component remoteConfigForNamespace:@"some_namespace"]);
 }
 
+- (void)testThrowsWithEmptyProjectID {
```

Comment:
The fact we're only asserting FIRRemoteConfigComponentTest.m throws supports the idea that validating inputs there is sufficient.